internal/backend/runtime/omni/controllers/omni/machine_set_status.go

```diff
@@ -125,6 +140,12 @@ func NewMachineSetStatusController() *MachineSetStatusController {
 			// config patch to machine set if the machine is allocated, checks by different layers, if is on the cluster layer,
 			// matches all machine sets
 			func(ctx context.Context, _ *zap.Logger, r controller.QRuntime, patch *omni.ConfigPatch) ([]resource.Pointer, error) {
+				if data := patch.TypedSpec().Value.GetData(); len(data) > 0 {
```

Comment:
I'm not sure this metric would work great way exactly, as it is skewed towards reconciles, and on no reconciles it would do exactly nothing, and prefer updated patches or non-updated. 

I don't think it's a great idea in this place.

Reply:
See how other metrics are done in controllers - if we _want_ to do this, we need a controller which every e.g. 30 seconds lists all config patches and updates the histogram.

Reply:
Done in https://github.com/siderolabs/omni/pull/979/files#diff-3a5c8c6112d9405bfd55d2edd58f32971d18564903cd232af42762ebfd4569f7R70